README.rst

```diff
@@ -58,6 +58,28 @@ The following configuration options are available:
 - `DRYDOCK_SENTRY_DSN` : The sentry DSN. Defaults to `""`.
 - `DRYDOCK_POD_LIFECYCLE` : Whether to enable pod lifecycle. Defaults to `true`.
 
+About Drydock v15.8.0
+---------------------
+
+This version of Drydock have some important features and improvements, is **highly** recommended to update to this version.
+
```

Comment:
We could replace this paragraph with a preface (tutor doesn't generate manifest files for jobs and in turn we can't use GitOps patterns, etc).